module/importer/src/Application/Model/Form/UploadModel.php

```diff
@@ -21,6 +21,7 @@ class UploadModel
 {
     private const FILE_EXTENSIONS = [
         'csv',
+        'zip',
```

Comment:
will this not make it possible to use a zip file, e.g. on other importers who do not have to support it ?

Reply:
We should set different `UploadModel` for each import :) I think that is quest for another task :)